build-ci/new_package.rst

```diff
@@ -21,8 +21,42 @@ then needs to be added to the :file:`ups/*.table` file (and possibly the
 :file:`ups/*.cfg` file) of one or more other packages in the stack where it is
 used.
 
+.. _lfs-repos:
+
+Handling ``git lfs`` backed repos
+=================================
+
+New `git lfs` (see :doc:`/tools/git_lfs`) backed repos (or existing repos
+being converted to `lfs`) require additional configuration.
+
+- The `repos.yaml`_ entry must declare that the repository is LFS backed.
+
+.. code-block:: yaml
+
+    afwdata:
+      url: https://github.com/lsst/afwdata.git
+      lfs: true
+
+  See the comment block at the top of `repos.yaml`_ for additional details.
+
+- At present, the EUPS `distrib` packaging mechanism does not support `lfs`
+  backed repos.  These products **must not** be added to any ``top`` level
+  meta-package or as a mandatory (non-``optional``) recursive dependency of a
+  ``top`` level package.
+
+- *Optional* dependencies must be added to `manifest.remap`_ to prevent the
+  creation of broken EUPS `distrib` packages.  Please note that the "self-merge"
+  policy of`RFC-75`_ does not apply to `manifest.remap`_.
+
+  *Unlike changes merged into `repos.yaml`_, modifications to `manifest.remap`_
+  do not take immediate affect*.  A manual deployment operation is required.  The
+  recommend procedure is to attach the modification PR to a DM Jira issue on the
+  ``Continuous Integration`` component.
+
```

Comment:
There's an issue with rst where you can't nest inline styles. So, the links here break because they're inside italics. A way to fix this is

```
*Unlike changes merged into* `repos.yam`_, *modifications to* `manifest.remap`_ *do not take immediate affect*
```